### ISSUE TITLE
Enhanced support for surface finish characteristics

### ISSUE THREAD

**Issue by GitHubDaveSmith** [QualityInformationFramework/qif-community#13]
More support is required for surface finish characteristics.

The element SurfaceTextureCharacteristicDefinition has the mandatory field RoughnessAverage (Ra) but there are many more surface finish characteristics that need representation. Depending on the parameter being evaluated there might be other parameters that need to be defined in addition to the surface finish characteristic. Examples are the type of filtering applied, use of the 'max' rule, short and long cut off values, the lay direction, the limit type (upper, lower or bilateral) a second target value for bilateral tolerances, bearing ratio etc. 

Another detail that a user might wish to see is the manufacturing method, for example grinding, honing, polishing, buffing etc.

**Comment by daniel-campbell** [comment]
Good point. I believe that Capvidia will be putting together some ideas for improvement in this area as well.

**Comment by larry-bergquist** [comment]
There has been changes in surface texture standards.
ASME Y14.36-2018 & B46.1-2019 are currently the latest from ASME.
![Snap19](https://user-images.githubusercontent.com/72532436/169870878-716c3e94-8c69-440d-8f87-9801934bc28c.jpg)

QIF member recruitment opportunity?
     Mark Malburg at Digital Metrology Solutions is very active in surface texture standards and the surface texture industry.
Maybe others as well?
